Clarify restrictions on cross-region PrivateLink Endpoints (#1682)
astronomer/docs: astro/connect-aws.md

<TabItem value="AWS PrivateLink">

> Use AWS PrivateLink to create private connections from Astro to your AWS services without exposing your data to the public internet.  PrivateLink endpoints can only be used to expose services in that were created in the same region as the Astro cluster.

Reviewer: ```suggestion
Use AWS PrivateLink to create private connections from Astro to your AWS services without exposing your data to the public internet.  PrivateLink endpoints can only be used to expose services that were created in the same region as the Astro cluster.
```

> Use AWS PrivateLink to create private connections from Astro to your AWS services without exposing your data to the public internet.  PrivateLink endpoints can only be used to expose endpoint services that were created in the same region as the Astro cluster.

Reviewer 2: The added sentence here is no longer true..

Reviewer: @jwitz fixed. This is easier when direct edits are provided.

Author: Could do with some help with this please. The above statement is true, there is just a work around that we can do if the upstream service is in a separate region. We are still exposing the service with the network load balancer in the same region as Astro, however that network load balancer is able to front a service that is in a different region.

Reviewer: @stuart23 if the workaround is completed by Astronomer support, we should probably add a statement similar to the following:

_If your AWS services are located in a different region than Astro, contact Astronomer support._

If this isn't accurate, feel free to provide alternate wording and I'll work with you to refine it.

Thanks!

Author: That sounds fine to me

Reviewer: @stuart23 the statement has been added. Thanks!

To request additional endpoints, or assistance connecting to other AWS services, contact [Astronomer support](https://cloud.astronomer.io/support).

> To access a service in a different region using PrivateLink endpoints, an inter-region VPC Peering Connection can be used to an intermediate VPC. The intermediate VPC can be created in the same region as the targetted service, and the PrivateLink endpoint can then be created in this VPC.

Reviewer: ```suggestion
To access a service in a different region using PrivateLink endpoints, you can use an inter-Region VPC peering connection. You create the inter-Region VPC peering connection in the same region as the targeted service, and then you create the PrivateLink endpoint in the VPC.
```
